tests/framework/pkg/environmentflag/environmentflags.go

```diff
@@ -11,6 +11,10 @@ const (
 	KubernetesUpgradeAllClusters EnvironmentFlag = iota
 	WorkloadUpgradeAllClusters
 	UpdateClusterName
+	Ingress
```

Comment:
Natalie just a really small thing here, I think while you're resolving the conflicts it picked both your changes and old ones. Ingress and Chart options are removed from the environment flags.

Reply:
I removed Charts and Ingress